requirements-dev.txt

```
@@ -0,0 +1,5 @@
+colorlog == 6.6.0
+pydantic ~=1.9.2
+python-logstash ~= 0.4.8
```

instead of the 3 lines 1-3 just put:
```suggestion
-r requirements.txt

```